config/site.php

```diff
@@ -426,6 +426,17 @@
                 'role' => __('CakePHP Trainer'),
                 'image-url' => 'team/jorge_gonzalez.jpg',
                 'country' => 'Spain'
+            ],
+			'garas' => [
```

Comment:
```suggestion
            'garas' => [
```

Use spaces instead of tabs.